eventmesh-connectors/eventmesh-openfunction-connector/src/main/java/org/apache/eventmesh/connector/openfunction/client/CallbackServiceGrpc.java

```diff
@@ -24,16 +41,16 @@ private CallbackServiceGrpc() {
 
     // Static method descriptors that strictly reflect the proto.
     @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
-    public static final io.grpc.MethodDescriptor<org.apache.eventmesh.common.protocol.grpc.cloudevents.CloudEvent,
+    public static final io.grpc.MethodDescriptor<CloudEvent,
```

Comment:
There are also substitutions in this class that can be made in this way.[line45,89,90,101,102,133,144...]